db/blip_sync_context.go

```diff
@@ -579,7 +579,7 @@ func (bsc *BlipSyncContext) sendBLIPMessage(sender *blip.Sender, msg *blip.Messa
 }
 
 func (bsc *BlipSyncContext) sendNoRev(sender *blip.Sender, docID, revID string, collectionIdx *int, seq SequenceID, err error) error {
-	base.DebugfCtx(bsc.loggingCtx, base.KeySync, "Sending norev %q %s due to unavailable revision: %v", base.UD(docID), revID, err)
+	base.TracefCtx(bsc.loggingCtx, base.KeySync, "Sending norev %q %s due to unavailable revision: %v", base.UD(docID), revID, err)
```

Comment:
Do we already log this information elsewhere, which is why we want to lower to Trace here?  I thought norevs were generally unexpected and merited the debug logging.

Reply:
Good catch - I had intended to move the debug logging up a level so we could get more contextual information about _why_ we're sending the norev (like in the `Active revision channels (%s) not in filtered channels (%s) for %q - sending original norev for %q` case)